### DISCUSSION TITLE
[1주차][온라인 스터디] 글 모음

### DISCUSSION THREAD

**Discussion by bigsaigon333** [fe-backpackers/fe-interview-basecamp#111] [General]
## 1주차 진행순서

~22:05 체크인(with github discussions)

~22:25 PR 리뷰

~22:40 입풀기(with 카훗)

~22:50 페어와 모의면접

~23:00 회고(스터디 목표, 스터디 좋았던 점, 건의사항, 체크아웃)

**Comment by bigsaigon333** [comment]
## 체크인

오늘의 기분을 10분 만점으로 표현해주세요

**Reply by bigsaigon333** [reply]
8점: 오늘 진짜 팀 프로젝트 이슈 처리한다고 정신없이 바빴고, 인터뷰도 하나 하고 왔습니다... 정신없지만 생각보다 많은 분들 와주셔서 너무 기분 좋습니다 :+1:

**Reply by ganeodolu** [reply]
8점 : 새로운 스터디 시작, 특히 인터뷰 관련이라 떨리네요ㅎㅎㅎ

**Reply by eotkd4791** [reply]
9점: 첫 스터디, 집중이 잘된 하루였습니다!

**Reply by ddongule** [reply]
10점: 오늘은 뭔가 기분이 좋네요 후하하항 스터디 재밌을 것 같아용 기대됩니다~~!!!!!!

**Reply by Lobo2u** [reply]
6점:백신 2차 휴유증으로 살짝 고생을 하긴 했지만 ㅠㅠ 그래두! 재미있는 스터디를 하게 되어서 기대가 됩니다! 숙제하면서 공부했던게 재미있긴했어요 ;) 아직은 깃허브에 서툰 초보긴하지만 열심히 공부하겠습니당

**Reply by jinsoobyeon** [reply]
10점: 드디어 백신 2차 맞았습니다!! ㅎㅎ

**Reply by jordan-choi** [reply]
8점: 어제는 병원 가느라 정신이 없었는데 오늘은 할일에 집중할 수 있어 좋았습니다ㅎㅎ

**Reply by ritajeong** [reply]
9점: 스터디 첫날 기대가 됩니당!!

**Reply by MINGDY98** [reply]
9점! 첫 모임이에요! 오늘 새로운 공부를 시작해서 기분이좋아요!:)

**Reply by ChaerinYu** [reply]
8점: 반갑습니다👏 모두 으쌰으쌰 화이팅 합시다

**Comment by bigsaigon333** [comment]
## 회고
- 스터디 목표
- 스터디 좋았던 점
- 건의사항
- 기타 등등

**Reply by ChaerinYu** [reply]
글로 쓰는 거 보다 직접 말로 하는 게 더 공부가 잘 되는 거 같아요! 다음 주는 더 열심히 준비해서 더 도움 되고 더 도움 받는 스터디가 되도록 하겠습니다 ㅎㅎ 😊

**Reply by jordan-choi** [reply]
* 목표: 알고있는 것을 명확하게 설명하는 것
* 좋았던 점: 2명 페어로 질문 + 시간 제한 두고 대답해보기 컨텐츠 좋았습니다. 다음 스터디 때 또 하면 좋을 것 같습니다.
* 건의사항은 없고, 다음 스터디 전까지 틈틈히 리뷰 남기겠습니다 ㅎㅎ

**Reply by Lobo2u** [reply]
아직 암기는 못하더라두 참여하면서 용어에 익숙해지다보면 다음에 보았을땐 좀 더 쉽게 개념 획득이 가능할것같아요!
무엇보다 꾸준히 공부를 할 수 있는 동기를 가질수 있는것이 좋은 것같습니다! 
아직은 모르는게 많지만 차근차근 제 목표를 향해 나아가겠습니다!

**Reply by bigsaigon333** [reply]
### 스터디 목표

내가 아는 지식을 말로 풀어서 할 수 있다 -> 그를 위한 수단으로서 글로 먼저 내 생각을 정리해본다

### 스터디 좋았던 점

페어로 서로 묻고 답할 때 다들 굉장히 몰입해서 하는 거 같아 보여서 좋았습니다.

저도 홀랄리님이 좋은 피드백 주셔서 좋았습니다. 그리고 홀라리님 굉장히 차분하게 말씀하시는데, 그런 톤, 어투 본받고 싶네요

### 건의사항

카훗 할 때 제대로 공지 못했던거 죄송해요 ㅠㅠ 다음에는 카훗 할 때 사용법 다시 공지하겠습니다 !

**Reply by jinsoobyeon** [reply]
- 스터디 목표: FE 개념 채득과 나만의 보이스로 설명
 - 좋았던 점: 정리하고 말로도 해보니까 확실히 부족한 부분이 보여서 보충할 수 있는 포인트를 알게 되는것 같아요!!
 - 건의사항: 없어요~

**Reply by ganeodolu** [reply]
- 스터디 목표 : 차근차근 공부한 것을 말할 수 있는 것
 - 스터디 좋았던 점 : 지금 상황을 실제로 면접에서 닥친다고 생각하니 아찔한 것 같습니다. 여기서 시뮬레이션 많이 할 수 있는 기회가 있어서 좋아요 😄

**Reply by ritajeong** [reply]
- 스터디 목표 : 확실히 말로 하는 게 글로 정리할 때와는 사뭇 다르다는 걸 알았어요 앞으로는 좀 더 자연스럽게 답변할 수 있도록, 매주 열심히 참여해보려고 합니다 목표는 올출석 👏
- 스터디 좋았던 점 : 페어님과 질의응답하면서 나누고 있었고, MDN에서 bigint가 원시타입으로 서술되어있었고 책과는 달라서 이유를 좀 찾아봐야겠다고 하던 중에, 책은 버전이 달라서 공식문서를 좀 더 기준을 두는 게 좋다고 조언해주셔서 너무나 감사했습니다 👍
- 건의사항 : 아직은 없습니닷!  on going 스터디라 더 좋은 것 같아요 앞으로가 기대됩니다 🎉

**Reply by ddongule** [reply]
### 스터디 목표
- 자바스크립트 기본 지식 다지기!

### 스터디 좋았던 점
- 혼자 글을 읽으며 머릿속으로 집어넣는 것 보다는 직접 말하면서 정리하는 게 더 머릿속에 쏙쏙 들어오는 것 같아서 좋았어요.
- KaHoot 재밌었어요! 문제 수를 더 늘려도 좋을 것 같습니당 ㅎㅎ
- 페어로 이야기 해 보는 거 좋았어욤!! 2명이 아니라 3명도 좋을 것 같아요 호호호 

### 체크아웃
- 10점! 베리굿! 재밌었어요! 같이 공부하니까 으쌰으쌰 힘이 나네욤

**Reply by eotkd4791** [reply]
* 스터디 목표
  * 면접에서 표현하고 싶은 것들을 잘 정리해서 답변하는 연습을 꾸준히 하고 싶습니다.
  
* 스터디 좋았던 점
  * 페어 모의면접과 피드백이 너무 좋았습니다.

* 건의사항
  * 서로의 PR을 리뷰하는 것도 스터디에서 매우 중요하다고 생각하는데 스터디 외 시간에 하는 게 더 효율적일 것 같습니다!

**Reply by MINGDY98** [reply]
### 스터디 목표 :
 javascript를 혼자 공부하는 것보다 더욱 효과적일 것이라 생각해 스터디에 참여하였습니다
### 스터디 좋았던 점 : 
실제로 개념을 말로 표현하고 나니 충분히 공부했다고 생각한 개념도 부족한 부분이 보였고, 더 많은 지식을 알 수 있었습니다. kahoot으로 랜덤주제에 대해 시간제한을 두고 말해보는 경험을 통해 실전대비를 할 수 있을 것같아 좋았고, 페어리뷰?는 특히 서로의 지식을 공유할 수 있어 좋았습니다.
### 건의사항 :
 이대로 계속 하면 정말 좋을 것 같습니다.

**Comment by bigsaigon333** [comment]
## 체크아웃

스터디 후 현재의 기분을 10점 만점으로 표현해주세요 :smile:

**Reply by ChaerinYu** [reply]
9점: 와아아아 🙌

**Reply by jinsoobyeon** [reply]
10점: 유익한 스터디!!

**Reply by Lobo2u** [reply]
7점:재미있게 잘하구갑니당! 다음주에 뵈요 ;)

**Reply by ddongule** [reply]
10점: 넘좋아용!!!!!!

**Reply by jordan-choi** [reply]
10점: 다음 스터디 기대됩니다!

**Reply by ritajeong** [reply]
10점: 오늘 하루는 살짝 놀았는데.. 하루의 마무리 시간에 이렇게 좋은 스터디하니까 뭐라도 배우고 가서 넘나 좋네요 👍

**Reply by MINGDY98** [reply]
10점 : 스터디 하고나니 더욱 잘하고 싶네요! 최고:)

**Reply by ganeodolu** [reply]
9점 : 스터디에 참여하길 잘했다는 생각이 들어요ㅎㅎ

**Reply by bigsaigon333** [reply]
9점: 몸은 피곤하지만 또 자극받고 갑니다! 화이팅~!
